app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -526,6 +526,10 @@ def build_toolbar_hide_button(id)
     return true if @record.kind_of?(OrchestrationStack) && @display == "instances" &&
                    %w(instance_check_compliance instance_compare).include?(id)
 
+    # hide select power management buttons for Openstack::InfraManager
+    return true if %w(host_standby host_shutdown host_reboot host_reset).include?(id) &&
+                   @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
```

Comment:
@romanblanco, @PanSpagetka I moved this code from toolbar_builder to the basic button visible? method. Part of the reason this was placed in toolbar_builder was to affect the visibility of the power management buttons in the hosts_center view in addition to the host_center view. Since hosts_center use the basic button and not the GenericFeatureButton, that seems to be the best place to put that piece of logic. I tried modifying hosts_center to also use GenericFeatureButton but it then complains about various validate_ methods like validate_standby is not defined in ManageIQ::Providers::Openstack::InfraManager. It seemed odd to add Hosts' validate_ methods into InfraManager.